shared-libs/cht-datasource/src/local/person.ts

```diff
@@ -17,7 +16,7 @@ export namespace v1 {
       return false;
     }
     const hasPersonType = contactTypeUtils.isPerson(settings.getAll(), doc);
-    if (!hasPersonType || !isNormalizedParent(doc)) {
```

Comment:
I got too scared of this `isNormalizedParent` check since it is validating the contents of `parent`.  I don't think, especially in `v1`, we want to block _reading_ a contact because of problems in their parent lineage.

Reply:
Also, it is worth noting that the original logic when reading a person in `shared-libs/contacts` was just to check the type and nothing else. So we are staying consistent here and avoiding new validations.